### ISSUE TITLE
does this hack work with Xiao Fang cameras?

### ISSUE THREAD

**Issue by pbizzlr** [fritz-smh/yi-hack#144]
like this one - https://www.aliexpress.com/item/EU-Edition-Xiaomi-Yi-Smart-Camera-2-Xiaoyi-ants-Home-Webcam-IP-camera2-wifi-wireless/32692695551.html?source=%7Bifdyn:dyn%7D%7Bifpla:pla%7D%7Bifdbm:DBM&albch=DID%7D&src=google&albch=shopping&acnt=494-037-6276&isdl=y&aff_short_key=UneMJZVf&albcp=658432961&albag=33762888219&slnk=&trgt=68416666751&plac=&crea=en32692695551&netw=g&device=c&mtctp=&gclid=CJvRi6_UxNICFQMM0wodoWMLTw ?

**Comment by SatoshiNakamoto-zz** [comment]
No, for XiaoFang use this! https://github.com/samtap/fang-hacks Works greate!

**Comment by yuyat** [comment]
https://github.com/fritz-smh/yi-hack/issues/118
